src/containers/help/HelpContainer.ts

```diff
@@ -0,0 +1,40 @@
+import { FC, useEffect, useState } from 'react';
+import { HelpApi } from '../../api/HelpApi';
+import { ComponentOrChildrenFn, renderComponentOrChildrenFn } from '../../utils/containers/ComponentOrChildrenFn';
+
+export interface HelpContainerProvided {
+  helpTextMd: string | undefined;
+  isLoading: boolean;
```

Comment:
It's probably a good idea to add error to be consistant.

Reply:
done